packages/babel-helper-define-polyfill-provider/src/node/dependencies.js

```diff
@@ -18,32 +18,15 @@ export function resolve(
     basedir = path.resolve(basedir, absoluteImports);
   }
 
-  let modulePackage, moduleNestedPath;
-
-  let slash = moduleName.indexOf("/");
-  if (moduleName[0] === "@") {
-    slash = moduleName.indexOf("/", slash + 1);
-  }
-
-  if (slash === -1) {
-    modulePackage = moduleName;
-    moduleNestedPath = "";
-  } else {
-    modulePackage = moduleName.slice(0, slash);
-    moduleNestedPath = moduleName.slice(slash);
-  }
-
   try {
-    let pkg;
     if (nativeRequireResolve) {
```

Comment:
Not related to this PR: we need to update `nativeRequireResolve` to use cheap semver so versions like `8.10` does not fall to `requireResolve`.